src/groovy/com/flashsales/session/SessionTrackerValve.groovy

```diff
@@ -0,0 +1,172 @@
+package com.flashsales.session
```

Comment:
Could you remove the reference to this package, it doesn't make sense in the context of this plugin.

Reply:
Done.